Formula/virtuoso.rb

```diff
@@ -36,19 +32,6 @@ class Virtuoso < Formula
 
   skip_clean :la
 
-  # Support OpenSSL 1.1
-  patch do
-    url "https://sources.debian.org/data/main/v/virtuoso-opensource/7.2.5.1+dfsg-2/debian/patches/ssl1.1.patch"
-    sha256 "9fcaaff5394706fcc448e35e30f89c20fe83f5eb0fbe1411d4b2550d1ec37bf3"
-  end
-
-  # TLS 1.3 compile error patch.
-  # This also updates the default TLS protocols to allow TLS 1.3.
-  patch do
-    url "https://github.com/openlink/virtuoso-opensource/commit/67e09939cf62dc753feca8381396346f6d3d4a06.patch?full_index=1"
-    sha256 "485f54e4c79d4e1e8b30c4900e5c10ae77bded3928f187e7e2e960d345ca5378"
-  end
-
   def install
     # We patched configure.ac on stable so need to rerun the autogen.
     system "./autogen.sh"
```

Comment:
Shouldn't be needed for stable anymore. We can restore head.
```suggestion
    system "./autogen.sh" if build.head?
```

Along with
```diff
@@ -13,11 +12,16 @@ class Virtuoso < Formula
     sha256 cellar: :any, high_sierra: "3abcc2f1444324d675af9014ac20555124c875d7e9a4ba9b021fd1ad7c570845"
   end

-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  head do
+    url "https://github.com/openlink/virtuoso-opensource.git", branch: "develop/7"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   # If gawk isn't found, make fails deep into the process.
   depends_on "gawk" => :build
-  depends_on "libtool" => :build
   depends_on "openssl@1.1"
```